src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,32 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
```

Comment:
The `Animal` constructor should not require a `health` parameter since health defaults to 100. Remove the unnecessary `health` parameter from the Animal constructor to adhere to the task requirements.